README.md

```diff
@@ -166,6 +166,9 @@ the one you are trying to build.
 -k, --keep
 if using docker, keep the container after the build.
 
+--make-exploded-image
+creates an exploded image (useful for codesigning jmods). Use --assemble-exploded-image once you have signed the jmods to complete the packaging steps.
```

Comment:
It's fine for the README. Ideally, there should be some sort of a recipe, too. I could not find info how to sign the jmods and our tooling isn't that approachable in general. Separate PR would be okay.